$OpenBSD: patch-libsrc_util_usage_c,v 1.1 2017/12/26 18:12:30 jca Exp $

Mark the default usage() implementation as weak to avoid multiple
definition errors with lld(1).

Index: libsrc/util/usage.c
--- libsrc/util/usage.c.orig
+++ libsrc/util/usage.c
@@ -28,8 +28,7 @@
 
 #include "libutil.h"
 
-
-void
+void __attribute__((weak))
 usage (void)
 {
   default_usage ();
